mysql-test/suite/tokudb.perfschema/t/start_server_tokudb-master.opt

```diff
@@ -0,0 +1 @@
+--loose-enable-performance-schema $TOKUDB_OPT $TOKUDB_LOAD_ADD --loose-tokudb-check-jemalloc=0 --loose-tokudb-cache-size=512M --loose-tokudb-block-size=1M
```

Comment:
Consider consolidating both option files to a suite.opt, it should work for any future Toku/PFS tests too